storybook/.storybook-react/webpack.config.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+const globImporter = require('node-sass-glob-importer');
+
+// Export a function. Accept the base config as the only param.
+module.exports = async ({ config, mode }) => {
+  // `mode` has a value of 'DEVELOPMENT' or 'PRODUCTION'
+  // You can change the configuration based on that.
+  // 'PRODUCTION' is used when building the static version of storybook.
+
+  // Adds SCSS support
+  config.module.rules.push({
+    test: /\.scss$/,
+    use: [
+      'style-loader',
+      {
+        loader: 'css-loader',
+        options: {
+          importLoaders: 1,
+          modules: true
+        }
+      },
+      {
+        loader: 'sass-loader',
+        options: {
+          importer: globImporter(), 
+        }
+      }
+    ],
+    include: [path.resolve(__dirname, './styles/**/*')]
+  });
+
+  // Add twig support
```

Comment:
This is no longer needed for the storybook-react since we have storybook-html, correct?

Reply:
Yeah, I could take this out.  I'll do that before merging